packages/js/src/ui/components/Notification/DefaultNotification.tsx

```diff
@@ -0,0 +1,9 @@
+import { InboxNotification } from '../../../types';
+
+type DefaultNotificationProps = {
+  notification: InboxNotification;
+};
+
+export const DefaultNotification = (props: DefaultNotificationProps) => {
+  return <>{props.notification.body}</>;
```

Comment:
❓ Is this where our default look-n-feel for notification item will live?

Reply:
Yes, creating this now

Reply:
Should we merge this PR and start a new one?

Reply:
Yeah, we'll do this since we're waiting on some possible design changes.